mix.exs

```diff
@@ -78,7 +78,13 @@ defmodule Membrane.VideoCompositor.Mixfile do
       links: %{
```

Comment:
I'm little bit confused: I thought version 0.1 was supposed to be based on Core 0.10. Are our users (notably the Live team) ok with Core 0.11?

Reply:
AFAIK they needed a released compositor compatible with core 0.11, what seems reasonable since they're currently moving to 0.11 ;)